### ISSUE TITLE
Indicate player turn and AI is slower as the game goes on

### ISSUE THREAD

**Issue by mikerichantk** [magliana22/BSGameActual#6]
I think the random AI might be checking random squares and when it checks a bunch of previously made moves it takes a while to get to a empty space to play.  One way around this would be making an array list of the board coordinates and then using Collection.shuffle() and then just iterate through the list so it always has its next move.

**Comment by dittowill123** [comment]
Yeah, that is an issue. Towards the end when most spaces have been fired at, it takes a while for the AI to keep checking whether a move is valid because of the random method in addition to the sleep function.
